api/app/Models/User.php

```diff
@@ -182,6 +182,7 @@ public function filterByPools(Builder $query, array $poolCandidates): Builder
                     } else if ($poolCandidates['expiryStatus'] == ApiEnums::CANDIDATE_EXPIRY_FILTER_EXPIRED) {
                         $query->whereDate('expiry_date', '<', date("Y-m-d"));
                     }
+                    PoolCandidate::poolCandidateStatus(['AVAILABLE']);
```

Comment:
We need to continue using the same queryBuilder object, and also use the statuses that were passed in the query.
```suggestion
                    $query->poolCandidateStatus($poolCandidates['statuses']);
```